### PR TITLE
[MM-54335] Ceiling tests improvements

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR makes further improvements necessary to complete our v1 ceiling tests. Most notably:

- Significantly bumped UDP listening sockets, given the existing values, was causing severe lock contention at scale.
- Added more metrics
  - Track write times
  - Track write packet drops
- Improved load estimation

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-54335

### CODE REVIEW

**Comment by streamer45** [comment]
I need to create a matching PR on the plugin side to make this pass but the changes are good to be reviewed.

File: service/rtc/net.go [hunk 1]
```diff
@@ -74,7 +81,7 @@ func getSystemIPs(log mlog.LoggerIFace, dualStack bool) ([]netip.Addr, error) {
 func createUDPConnsForAddr(log mlog.LoggerIFace, network, listenAddress string) ([]net.PacketConn, error) {
 	var conns []net.PacketConn
 
-	for i := 0; i < runtime.NumCPU(); i++ {
+	for i := 0; i < getUDPListeningSocketsCount(); i++ {
```
Comment (cpoile) [inline service/rtc/net.go:84]: oh no, this is going to spam darwin systems on startup. 😛

File: service/system.go [hunk 2]
```diff
@@ -6,14 +6,48 @@ package service
...
+			if err != nil {
+				s.log.Error("failed to get cpu stat", mlog.Err(err))
+				continue
+			}
+
+			idleDiff := currStat.CPUTotal.Idle - prevStat.CPUTotal.Idle
+
+			if !prevTime.IsZero() {
+				s.mut.Lock()
+				s.systemInfo.CPULoad = 1 / (idleDiff / currTime.Sub(prevTime).Seconds())
```
Comment (cpoile) [inline service/system.go:41]: This is to account for ticker skew/delay, or something else?

Comment (streamer45) [reply service/system.go:41]: Yes, it's normalizing it against the actual time spent.

**Pull request merged**